ads/sovrn.js

```diff
@@ -16,19 +16,21 @@
  * Existing sovrn customers feel free to contact amp-implementations@sovrn.com for assistance with setting up your amp-ad tagid
  * New customers please see www.sovrn.com to sign up and get started!
  *********
```

Comment:
Could you please move this paragraph out of license and into a separate comment?